kolibri/core/hooks.py

```diff
@@ -0,0 +1,40 @@
+"""
+Kolibri Core hooks
+------------------
+
+WIP! Many applications are supposed to live inside the core namespace to make
+it explicit that they are part of the core.
+
+Do we put all their hooks in one module or should each app have its own hooks
+module?
```

Comment:
> Oh and I also think it would be good to separate the "framework" code from the "usage" code. In other words, abstract hook stuff should probably live separately from even core concrete uses of it.

Agreed. `hooks.py` should define the "abstract hooks of this application which others can hook into" or something.. again terminology seems to be a delicate thing :)